### PR TITLE
Update change-the-default-language-for-greetings-and-emails.md

### PR DESCRIPTION
**Pull request by carolynblanding**
@CarolynRowe I spoke with @roykuntz  about this and reviewed proposed changes. Changes made here to provide correct instructions for Tenant admins to manage user preferred language settings with links to relevant documentation and to add link to instructions for users to change settings. Also make a tweak change settings to greetings to be relevant to this topic. 

Let me know if you have any questions. Thanks.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/change-the-default-language-for-greetings-and-emails.md [hunk 1]
```diff
@@ -33,14 +33,14 @@ If you are a [global administrator](https://support.office.com/article/da585eea-
   
 - **The languages that are available to you are determined by the location of your organization**. For example, if your organization is located in the United States, you can set the default language to English or Spanish. If your organization is located in Canada, you can choose between English and French. For a list of supported languages, see [Languages for voicemail greetings and messages from Skype for Business](languages-for-voicemail-greetings-and-messages.md).
     
-- **There's no way to change the system language for only one person in your organization.** You can only change the greeting language for everyone on your organization.
+- **Changing languages for individual user's voicemail greeting and voicemail messages.** You can change the preferred lanaguage for users, which will change the language of the their voicemail greeting and voicemail messages sent to their Outlook mailbox. For instruction please see [How to set language and region settings for Office 365] (https://docs.microsoft.com/en-us/office365/troubleshoot/access-management/set-language-and-region).
```
Comment (JohanFreelancer9) [inline Teams/change-the-default-language-for-greetings-and-emails.md:36]:
```suggestion
- **Changing languages for individual user's voicemail greeting and voicemail messages.** You can change the preferred language for users, which will change the language of their voicemail greeting and voicemail messages sent to their Outlook mailbox. For instructions, please see [How to set language and region settings for Office 365] (https://docs.microsoft.com/office365/troubleshoot/access-management/set-language-and-region). 
```

File: Teams/change-the-default-language-for-greetings-and-emails.md [hunk 2]
```diff
@@ -33,14 +33,14 @@ If you are a [global administrator](https://support.office.com/article/da585eea-
...
     
-- **There's no way to change the system language for only one person in your organization.** You can only change the greeting language for everyone on your organization.
+- **Changing languages for individual user's voicemail greeting and voicemail messages.** You can change the preferred lanaguage for users, which will change the language of the their voicemail greeting and voicemail messages sent to their Outlook mailbox. For instruction please see [How to set language and region settings for Office 365] (https://docs.microsoft.com/en-us/office365/troubleshoot/access-management/set-language-and-region). 
     
     > [!NOTE]
-    > Users can change their own greeting language through their settings after they sign in. 
+    > Users can change their own greeting language through their settings after they sign in by following instructions found in [Change your display language and time zone in Microsoft 365 for Business] (https://support.office.com/en-us/article/change-your-display-language-and-time-zone-in-microsoft-365-for-business-6f238bff-5252-441e-b32b-655d5d85d15b?ui=en-US&rs=en-US&ad=US)
```
Comment (JohanFreelancer9) [inline Teams/change-the-default-language-for-greetings-and-emails.md:39]:
```suggestion
    > Users can change their own greeting language through their settings after they sign in by following instructions found in [Change your display language and time zone in Microsoft 365 for Business] (https://support.office.com/article/change-your-display-language-and-time-zone-in-microsoft-365-for-business-6f238bff-5252-441e-b32b-655d5d85d15b?ui=en-US&rs=en-US&ad=US).
```

File: Teams/change-the-default-language-for-greetings-and-emails.md [hunk 3]
```diff
@@ -33,14 +33,14 @@ If you are a [global administrator](https://support.office.com/article/da585eea-
...
   
 - **Do you want to record your outgoing voicemail message?** See [Check Skype for Business voicemail and options](https://support.office.com/article/2deea7f8-831f-4e85-a0d4-b34da55945a8). For Microsoft Teams - Users can change their voicemail settings from the [Teams desktop client settings](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f)
```
Comment (JohanFreelancer9) [inline Teams/change-the-default-language-for-greetings-and-emails.md:41]:
```suggestion
- **Do you want to record your outgoing voicemail message?** See [Check Skype for Business voicemail and options](https://support.office.com/article/2deea7f8-831f-4e85-a0d4-b34da55945a8). For Microsoft Teams - Users can change their voicemail settings from the [Teams desktop client settings](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f).
```

File: Teams/change-the-default-language-for-greetings-and-emails.md [hunk 4]
```diff
@@ -33,14 +33,14 @@ If you are a [global administrator](https://support.office.com/article/da585eea-
...
 
-- **Do you want to change the voicemail prompt language?** For Skype for Business -  [https://mysettings.lync.com/voicemail](https://mysettings.lync.com/voicemail) and choose a new language under **Prompt Language**. For Microsoft Teams - Users can change their voicemail settings from the [Teams desktop client settings](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f)
+- **Do you want to change the voicemail prompt language?** For Skype for Business -  [https://mysettings.lync.com/voicemail](https://mysettings.lync.com/voicemail) and choose a new language under **Prompt Language**. For Microsoft Teams - Users can change their voicemail greeting from the [Teams desktop client settings](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f)
```
Comment (JohanFreelancer9) [inline Teams/change-the-default-language-for-greetings-and-emails.md:43]:
```suggestion
- **Do you want to change the voicemail prompt language?** For Skype for Business - go to the [User Settings Portal](https://mysettings.lync.com/voicemail) and choose a new language under **Prompt Language**. For Microsoft Teams - Users can change their voicemail greeting from the [Teams desktop client settings](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f).
```

**Pull request merged**